packages/nextjs/pages/index.tsx

```diff
@@ -48,6 +48,31 @@ const Home: NextPage<{
         </div>
       </div>
 
+      {/* Quote section*/}
+      <div className="bg-base-100">
+        <div className="container max-w-[90%] lg:max-w-7xl m-auto py-16 lg:py-20 lg:pl-20 flex flex-col-reverse lg:flex-row items-center lg:items-start lg:gap-10">
+          {/* Quote image */}
+          <div className="flex flex-col items-center max-w-md lg:max-w-md lg:mt-6">
+            <Image
+              src="/assets/austin-pfp.png"
+              alt="austin griffith"
+              width={120}
+              height={120}
+              className="rounded-[150px] shadow-xl"
+            />
+          </div>
+          {/* Quote Text */}
+          <div className="md:max-w-[80%] lg:max-w-[70%] flex flex-col items-center lg:items-start">
+            <p className="font-thin text-2xl lg:text-2xl text-center lg:text-left mb-10 lg:mb-4">
+              “The BuidlGuidl is a curated group of developers and designers with the goal of on boarding new developers
```

Comment:
"on boarding" is ok? Or should be "onboarding"? I usually see "onboarding" even tho the corrector doesn't seem to like it.

Reply:
Yeah true, I also think it should be onboarding. Good call!